src/fiber/ReactHardwareFiberComponent.js

```diff
@@ -0,0 +1,119 @@
+/**
```

Comment:
Note: separation between `ReactDOMFiber` and `ReactDOMFiberComponent` is just to keep `ReactDOMFiberComponent` similar to `ReactDOMComponent` (Stack) so we can mirror changes in both while they're forked. There's no special reason why separating these two files is a good idea.